accounts/migrations/0008_auto_20170331_1702.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hmmm good question. It went as follows: 
1) I checked out donations popup branch and merged master into it because I knew there were db changes and otherwise I could not work on donations popup branch without reverting db changes locally
2) Django complained there were differences with the models and db so I run make migrations and generated this file. This should have been run while I was on the other branch and commit there, but I guess I mis-thought this was because some side-effect of the merge of master to donations popup branch and I run it here and committed/pushed.

So human mistake here. However to me this raises interesting questions: how to deal with branches with different db "versions"?